optional_plugins/README.rst

```diff
@@ -0,0 +1,26 @@
+==================
+ Optional Plugins
+==================
+
+This is where plugins shipped with Avocado, but not considered core
```

Comment:
Well, seems like we are already using that naming. So I think you could just use: "... but not considered core plugins, can be found."